packages/core/src/browser/icons/expand.svg

```diff
@@ -2,3 +2,4 @@
 <!--Copyright (C) 2019 TypeFox and others.-->
 <!--Licensed under the MIT License. See License.txt in the project root for license information.-->
 <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path fill="#646465" d="M11 10.07h-5.656l5.656-5.656v5.656z"/></svg>
+
```

Comment:
It looks-like this icon was not updated, other than a newline being added at the end. Could you revert the change?

Reply:
How do I do that theia keeps adding the final newline

Reply:
I guess auto-formatting is kicking-in and adding the newline at the end. No big deal - we can keep it.

Reply:
k